saltproc/tests/test_depcode.py

```diff
@@ -12,25 +14,57 @@
 # global class object
 directory = os.path.dirname(path)
 
+# Serpent initalization
 serpent = DepcodeSerpent(
     exec_path='sss2',
-    template_inputfile_path=directory +
+    template_inputfiles_path=directory +
     '/template.inp',
-    iter_inputfile=directory +
-    '/test',
-    iter_matfile=directory +
-    '/material',
     geo_files=[
         os.path.join(
             directory,
             '../test_geo.inp')])
+serpent.iter_inputfile = directory + '/test'
+serpent.iter_matfile = directory + '/material'
+geo_test_input = directory + '/test_geometry_switch.inp'
+
+# Openmc initlialization
+openmc_input_path = os.path.join(directory, 'test_data/openmc/')
+openmc_test_inputfiles = {
+    "materials": "test_materials.xml",
+    "geometry": "test_geometry.xml",
+    "settings": "test_settings.xml",
+    "chain_file": "test_chain.xml"
+}
+
+openmc_iter_inputfiles = {
+    "geometry": "geometry.xml",
+    "settings": "settings.xml",
+}
+
+# prepend the correct paths to our
+# template and iter input files
+for key in openmc_test_inputfiles:
+    openmc_test_inputfiles[key] = os.path.join(
+        openmc_input_path, openmc_test_inputfiles[key])
+for key in openmc_iter_inputfiles:
+    openmc_iter_inputfiles[key] = os.path.join(
+        directory, openmc_iter_inputfiles[key])
+
+openmc = DepcodeOpenMC(
+    exec_path='../openmc_deplete.py',
+    template_inputfiles_path=openmc_test_inputfiles,
+    geo_files=[
+        os.path.join(
+            openmc_input_path,
+            'test_geometry_switch.xml')])
+openmc.iter_inputfile = openmc_iter_inputfiles
+openmc.iter_matfile = directory + '/materials.xml'
+
 
 msr = Reactor(volume=1.0,
               power_levels=[1.250E+09, 1.250E+09, 5.550E+09],
               dep_step_length_cumulative=[111.111, 2101.9, 3987.5])
 
-geo_test_input = directory + '/test_geometry_switch.inp'
-
 
```

Comment:
Functions missing docstrings (pre-existing functions, so possibly not intended to have docstrings?)

Reply:
Correct. Many of the test functions do not have docstrings. There is an open issue to fix this #39